mlrun/utils/notifications/notification_pusher.py

```diff
@@ -232,13 +234,15 @@ def _push_notification_sync(
                 exc=mlrun.errors.err_to_str(exc),
                 traceback=traceback.format_exc(),
             )
+            update_notification_status_kwargs["reason"] = f"Exception error: {str(exc)}"
```

Comment:
I'm a bit concerned since the `reason` field in the DB is 255 chars long, and we don't have any guarantee on exception str representation length. I guess in most cases this won't be an issue, but I would make sure to limit it (probably best to do in the `_update_notification_status` method since all code flows through there).